HPCC-14955 DOCS:Remove Auto Discovery

g-pan opened the pull request:
Fix HPCC-14955 DOCS:Remove Auto Discovery
Remove references to Auto discovery from the actual
text in the docs. Image can not be changed at this time
since the image is as the interface.

@JamesDeFabia please review 

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

<!-- Thank you for submitting a pull request to the HPCC project

 PLEASE READ the following before proceeding.

 This project only accepts pull requests related to open JIRA issues.
 If suggesting a new feature or change, please discuss it in a JIRA issue first.
 If fixing a bug, there should be an issue describing it with steps to reproduce.
 The title line of the pull request (and of each commit within it) should refer to the
 associated issue using the format:

 HPCC-nnnnn Short description of issue

 This will allow the Jira ticket to be automatically updated to refer to this pull request,
 and will ensure that the automatically-generated changelog is properly formatted.
 Where a pull request contains a single commit the pull request title will be set automatically,
 assuming that the commit has followed the proper guidelines.

 Please go over all the following points, and put an `x` in all the boxes that apply. You may find
 it easier to press the 'Create' button first then click on the checkboxes to edit the comment.
-->

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browers
  - [ ] The component(s) render as expected

## Testing:
<!-- Please describe how this change has been tested.-->
Local build, no issues. 
<!-- Thank you for taking the time to submit this pull request and to answer all of the above-->

g-pan committed:
HPCC-14955 DOCS:Remove Auto Discovery

Fix HPCC-14955 DOCS:Remove Auto Discovery
Remove references to Auto discovery from the actual
text in the docs. Image can not be changed at this time
since the image is as the interface.

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

HPCCSmoketest commented:
Automated Smoketest: :white_check_mark:
Sha: ae2468f1f7ccd557a9ece7f295e96c7fa43bbad4
Build: success
ECL Watch:  Rebuilding Site

| errors | warnings | build time | 
|---|---|---|
| 0 | 65 | 91.022 seconds |


Install hpccsystems-platform-community_6.2.1-documentation0.el7.x86_64.rpm
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | 
|---|---|---|---|---|---|
|unittest |88 |88 |0 |0 |0 |



HPCC Stop: OK
HPCC Uninstall: OK

g-pan committed:
HPCC-14955 DOCS:Remove Auto Discovery

Fix HPCC-14955 DOCS:Remove Auto Discovery
Remove references to Auto discovery from the actual
text in the docs. Image can not be changed at this time
since the image is as the interface.

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

HPCCSmoketest commented:
Automated Smoketest: :x:
Sha: 440ce335bd904ff8491b14c878b3db919cea5836
Build: failed
Number of Error(s): 2
  Fatal Error: download of
-- Configuring incomplete, errors occurred!
Get ECLWatch build result

JamesDeFabia reviewed (approved):
Looks good but please look into smoke test failure

HPCCSmoketest commented:
Automated Smoketest: :white_check_mark:
Sha: 440ce335bd904ff8491b14c878b3db919cea5836
Build: success
ECL Watch:  Rebuilding Site

| errors | warnings | build time | 
|---|---|---|
| 0 | 65 | 92.475 seconds |


Install hpccsystems-platform-community_6.2.1-documentation0.el7.x86_64.rpm
HPCC Start: OK

Unit tests result:

| Test | total | passed | failed | errors | timeout | 
|---|---|---|---|---|---|
|unittest |88 |88 |0 |0 |0 |



HPCC Stop: OK
HPCC Uninstall: OK
